app/invocation/cache_requests_card.tsx

```diff
@@ -1129,3 +1183,8 @@ function groupResults(
 function looksLikeDigest(actionId: string | null) {
   return actionId?.length === 64;
 }
+
+function redactInvocationId(invocationId: string) {
+  if (invocationId.length <= 8) return invocationId;
+  return `${invocationId.slice(0, 4)}..${invocationId.slice(-4)}`;
```

Comment:
nit: using ellipsis here would be a little nicer / more conventional than two periods `..`

```suggestion
  return `${invocationId.slice(0, 4)}…${invocationId.slice(-4)}`;
```